ronaldo-alves/1.I/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Sistemas de informação</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+   <h1>Sistemas de informação Home Page</h1>
+   <h2>Sistemas de informação</h2>
```

Comment:
Is there any reason to have added a duplicated title for "Sistemas de informação"?